tests/neo-vm.Tests/UtScriptBuilder.cs

```diff
@@ -61,6 +61,12 @@ public void TestEmitSysCall()
                 Assert.ThrowsException<ArgumentNullException>(() => script.EmitSysCall(null));
             }
 
+            using (var script = new ScriptBuilder())
+            {
+                script.EmitSysCall("Neo.Runtime.GetTrigger", true);
+                CollectionAssert.AreEqual(new byte[] { (byte)OpCode.SYSCALL, 0x04, 0x75, 0xC8, 0x93, 0xE3 }.ToArray(), script.ToArray());
+            }
+
```

Comment:
Good test here! But man, stop being lazy and add another test with the complete byte array and passing all bytes manually xD